### ISSUE TITLE
PopOverWidget doesn't work for satellite 6.9 settings page.

### ISSUE THREAD

**Issue by jameerpathan111** [SatelliteQE/airgun#546]
In Satellite 6.9, the UI for editing/viewing settings field is changed.
Refer:
https://github.com/SatelliteQE/airgun/blob/master/airgun/widgets.py#L2125
https://github.com/SatelliteQE/airgun/blob/master/airgun/views/settings.py#L16

**Comment by tpapaioa** [comment]
In 6.9.0 (snap 7.0), the Settings table looks like this:

```
<div class="tab-pane active" id="General">
  <foreman-react-component name="SettingsTable" data-props="{&quot;category&quot;:&quot;Setting::General&quot;}">
[...]
  <tr>
    <td class=""><span>Administrator email address</span></td>
    <td class=""><span><span class="editable">root@example.com</span></span></td>
    <td class="">The default administrator email address</td>
  </tr>
```

Compare to 6.8.2 (snap 4.0):

```
<div class="tab-pane active" id="General">
  <table class="table table-bordered table-striped table-hover ">
[...]
    <tr>
      <td><span data-original-title="administrator" rel="twipsy">Administrator email address</span></td>
      <td class="setting_value"><span class="editable editable-click" title="" data-type="text" data-model="setting" data-name="value" data-value="root@example.com" data-placeholder="root@example.com" data-url="/settings/administrator" data-original-title="Administrator email address (Default: root@example.com)">root@example.com</span></td>
      <td>The default administrator email address</td>
    </tr>
```

**Comment by mshriver** [comment]
Should be resolved by #545
